modules/gui/view_logs.py

```diff
@@ -0,0 +1,94 @@
+from modules.app.helper import Vector2
+
+from modules.gui.gui_module import GuiModule
+
+from tkinter import *
+from tkinter.font import BOLD
+from tkinter.scrolledtext import ScrolledText
+
+from modules.logging import Logging
+
+class GUI_ViewLogs( GuiModule ):
+
+    def drawLog( self, frame ):
+        header_frame = Frame(frame)
+        header_frame.pack(side=TOP, fill='x', pady=0, padx=(0,20))
+
+        table = Frame(header_frame)
+        table.pack(fill=BOTH, expand=1)
+
+        # Create the columns
+        ip_label = Label(table, text="IP", width=15)
+        ip_label.grid(row=0, column=0)
+        date_label = Label(table, text="Date", width=15)
+        date_label.grid(row=0, column=1)
+        content_label = Label(table, text="Content", width=30)
+        content_label.grid(row=0, column=2)
+
```

Comment:
It would be handy to also be able to have a column for the file; currently, when a file is created, the UI only shows the IP address, “Created” for the comment, and the date and time—it is not clear which file was created.
(Although there is the question of whether there is space for another column. Another option could be to dynamically add the filename to the comment for entries that do not already have the filename in the comment, using the `entry["file"]` that’s already included in each log entry.)